server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/CopyDefinitionNode.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree;
+
+import com.google.common.collect.ImmutableList;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.eclipse.lsp.cobol.core.model.Locality;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** The class represents the CopyBookDefinition Node. */
+@ToString(callSuper = true)
+@Getter
+public class CopyDefinitionNode extends Node {
```

Comment:
I'm not sure that it should extend Node when it is not going to be built in the tree structure.

Reply:
Yeah, you are right. We already got the `Context` interface which returns a list of Nodes for definitions and usages. So far every definition and usage was a part of the tree. That is a good catch.